app/src/main/java/com/example/plantsapp/MainActivity.kt

```diff
@@ -2,12 +2,40 @@ package com.example.plantsapp
 
 import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import androidx.fragment.app.Fragment
+import com.example.plantsapp.databinding.ActivityMainBinding
 
 class MainActivity : AppCompatActivity() {
 
+    private lateinit var binding: ActivityMainBinding
```

Comment:
1. For now you can remove this property from MainActivity and just save it inside of onCreate - because it is not used anywhere.